nilearn/tests/test_masking.py

```diff
@@ -16,11 +16,10 @@
 from nilearn.masking import (compute_epi_mask, compute_multi_epi_mask,
                              compute_background_mask, unmask, _unmask_3d,
                              _unmask_4d, intersect_masks, MaskWarning)
+from nilearn._utils.testing import (write_tmp_imgs, assert_raises_regex,
+                                    assert_warns_regex)
 
-from nilearn._utils.testing import write_tmp_imgs, assert_raises_regexp
-
-np_version = (np.version.full_version if hasattr(np.version, 'full_version')
-              else np.version.short_version)
+np_version = getattr(np.version, 'full_version', np.version.short_version)
```

Comment:
I'll revert this back to untouched; I thought it was causing an error from a merge conflict, but turns out it was in an import above.